### ISSUE TITLE
Question about uploading excel files

### ISSUE THREAD

**Issue by csandreas1** [google/google-api-php-client#1400]
Hello, can this library be used to upload form data on excel file on google drive or there is an alternative library for this?

**Comment by mikemcmullan** [comment]
Yes, you can use this library to upload excel files then convert them to Google Sheets if you want.
